src/screens/CostsListScreen/CostsListScreen.tsx

```diff
@@ -1,37 +1,40 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { Platform, Text, View } from 'react-native';
 
 import { StackNavigationOptions } from '@react-navigation/stack';
 import { HeaderButtons, Item } from 'react-navigation-header-buttons';
 
 import HeaderButton from '~/components/HeaderButton';
+import { useAppTranslation } from '~/hooks/useAppTranslation';
 import { NavigationData } from '~/navigation/AppNavigator/CostsNavigator';
 
 type Props = NavigationData<'CostsList'>;
 
-const CostsListScreen: React.FC<Props> = () => {
+const CostsListScreen: React.FC<Props> = ({ navigation }) => {
+  const { t } = useAppTranslation();
+
+  useEffect(() => {
```

Comment:
This part (getting translated text and setting header title, icon) could potentially repeat in a couple of screens in the future.

I could suggest creating a custom hook that does exactly this. Something like: `useTranslatedScreenTitle(titleKey, otherKey, headerIcon)`.

It would utilize `useAppTranslation`, `useEffect` and `useNavigation` (from `react-navigation`) under the hood. Have a look and let me know your opinion, maybe it would be possible 😀 I can explain it in more detail if needed.

Reply:
After discussion we are not going to do that for now.